Gemfile.lock

```diff
@@ -27,47 +27,31 @@ PATH
 GEM
```

Comment:
I guess that for an application you would package the entire bundle, and in that case you still wouldn't need the lock file. You would build your application and package the "binaries".

But, anyway, this is a plugin for vagrant (which in turn is an application). Right?